docs/pycco.css

```diff
@@ -1,3 +1,19 @@
+/**
```

Comment:
that is not our code at all :)

Reply:
Then we must somehow trick the license script into skipping this file.

Reply:
yeah

Reply:
@natiiix IMHO the script should accept any license, because we might have this problem with other files as well.

Reply:
fixed and created an issue to not have this problem in the future https://github.com/RedHatInsights/insights-results-aggregator/issues/885